airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/data/AirbyteValue.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.data
+
+sealed interface AirbyteValue
+
+data object NullValue : AirbyteValue
+
+@JvmInline value class StringValue(val value: String) : AirbyteValue
+
+@JvmInline value class BooleanValue(val value: Boolean) : AirbyteValue
+
+@JvmInline value class IntegerValue(val value: Long) : AirbyteValue
+
+@JvmInline value class NumberValue(val value: Double) : AirbyteValue
```

Comment:
```suggestion
@JvmInline value class NumberValue(val value: BigDecimal) : AirbyteValue
```

(we've seen loss of precision with float64)

Reply:
fixed